### PR TITLE
0.85/wakeup

### PR DESCRIPTION
**Pull request by orthagh**
Proposal for #57

### CODE REVIEW

File: inc/agentwakeup.class.php [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+   GNU Affero General Public License for more details.
+
+   You should have received a copy of the GNU Affero General Public License
+   along with FusionInventory. If not, see <http://www.gnu.org/licenses/>.
+
+   ------------------------------------------------------------------------
+
+   @package   FusionInventory
+   @author    David Durieux
```
Comment (ddurieux) [inline inc/agentwakeup.class.php]: I'm not the author of this file ;)

File: inc/timeslot.class.php [hunk 2]
```diff
@@ -138,7 +138,7 @@ function getTimeslotEntries($timeslot_ids = array(), $weekdays = null ) {
       if ( !is_null($weekdays) ) {
          $condition[] = "and `day` = '".$weekdays."'";
       }
-
+
```
Comment (ddurieux) [inline inc/timeslot.class.php]: Remove these spaces

**Comment by ddurieux** [comment]
Please add a test for check if cron is present in install and update test (you have yet other cron check) because seems not created on install

**Comment by orthagh** [comment]
ok, corrections soon

**Comment by ddurieux** [comment]
Rebased + merged manually

**Pull request closed without merging**